azurelinuxagent/ga/cgroupapi.py

```diff
@@ -42,6 +42,28 @@
 EXTENSION_SLICE_PREFIX = "azure-vmextensions"
 
 
+def is_systemd_failure(unit_name, stderr):
```

Comment:
move to azurelinuxagent/common/osutil/systemd.py and rename to is_systemd_run_failure